File: tempWuServer.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import urllib

link = "https://www.foto-webcam.eu/webcam/luftschiff/?frame=1"
f = urllib.urlopen(link)
myfile = f.read()

dateBeginStr = "\"date\":\"";
dateBegin = myfile.find(dateBeginStr);
dateEnd = myfile.find("\",", dateBegin);
readDate = myfile[dateBegin + len(dateBeginStr):dateEnd];

tempBeginStr = "\"wx\":\"";
tempBegin = myfile.find(tempBeginStr);
tempEnd = myfile.find("\",", tempBegin);
readTemp = myfile[tempBegin + len(tempBeginStr):tempEnd];
readTemp = readTemp.replace("\u00b0", "º");

print(readDate + ";" + readTemp);

File: tempWuServer.php
<?php

	$link = 'https://www.foto-webcam.eu/webcam/luftschiff/?frame=1';
	$f = file_get_contents($link);
	$dateBeginStr = '"date":"';
	$dateBegin = strpos($f, $dateBeginStr);
	$dateEnd = strpos($f, '",', $dateBegin);

	$readDate = substr($f, ($dateBegin + strlen($dateBeginStr)), $dateEnd - ($dateBegin + strlen($dateBeginStr)));

	$tempBeginStr = '"wx":"';

	$tempBegin = strpos($f, $tempBeginStr);
	$tempEnd = strpos($f, '",', $tempBegin);

	$readTemp = substr($f, ($tempBegin + strlen($tempBeginStr)), $tempEnd - ($tempBegin + strlen($tempBeginStr)));
	$readTemp = str_replace('\u00b0', 'º', $readTemp);

	echo $readDate . ';' . $readTemp;

?>